### PR TITLE
Add complete and reopen admin task functionality from People/new hire view

### PR DESCRIPTION
**Pull request by JonasMe**
Added "Action" header to "Admin task" table in "new hire -> Admin task" view.
image

Upon clicking the task will either be completed or reopened and appropiate message displayed.

image

### CODE REVIEW

**Review by GDay — COMMENTED** [review]
Thanks for the PR @JonasMe! I like the feature of allowing users to mark admin tasks from the overview view instead of having to go into the admin task to make it as completed.

I have my doubts about the reopening of tasks. As you saw, the admin tasks trigger other conditions from a sequence. The issue here is that not all of these triggers are logged/saved. Some of these (like the integration items) run and are not assigned to a user in any easy to lookup way, which means that completing the same admin task again would rerun that integration, which is likely not the expected behavior. So I would prefer to not allow the reopening of tasks at the moment.

**Comment by JonasMe** [comment]
Hi @GDay  - Thanks for the feedback. I can see the sense in removing the "reopening" until perhaps there is a lookup-table of executed tasks to prevent rerunning.

I'll remove that option as well as fixing the tests asap.

File: back/admin/people/new_hire_views.py [hunk 1]
```diff
@@ -702,3 +702,25 @@ class NewHireDeleteView(
     queryset = get_user_model().new_hires.all()
     success_url = reverse_lazy("people:new_hires")
     success_message = _("New hire has been removed")
+
+
+class CompleteAdminTaskView(
```
[the remark this answers is not in the document]
Comment (JonasMe) [reply back/admin/people/new_hire_views.py:707]: Thanks for the feedback @GDay 

I've seen that view but i guess i'm still figuring the conventions revolving the repo.

How would you prefer i reuse the view, while at the same time making sure the redirect goes to the right page (and not the admin task detail page)?

Thanks a bunch

[the remark this answers is not in the document]
Comment (GDay) [reply back/admin/people/new_hire_views.py:707]: Ah, that's right. I forgot about the redirect. I think this is fine for now. Thanks!

File: back/admin/people/templates/_unmark_complete_icon.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ <svg xmlns="http://www.w3.org/2000/svg" style="margin: 0 -0.25rem 0 -.25rem" class="icon icon-tabler icon-tabler-discount-off" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
```
Comment (GDay) [inline back/admin/people/templates/_unmark_complete_icon.html:1]: I think we can remove this file as well now since the removing feature has been removed.

**Review by GDay — CHANGES_REQUESTED** [review]
Two small things that would be nice to address and then this looks good to merge. Thanks!

**Review by GDay — APPROVED** [review]
LGTM! Thanks!

**Comment by JonasMe** [comment]
> LGTM! Thanks!

Great to hear @GDay  - a whitespace was missing in the auth files which Black dident like. I've reformatted and all should be good to go.

**Review by GDay — APPROVED** [review]

**Pull request merged**